stable/minio/values.yaml

```diff
@@ -123,14 +123,34 @@ gcsgateway:
   # Google cloud project-id
   projectId: ""
 
+## Use minio on NAS backend
+## https://docs.minio.io/docs/minio-gateway-for-nas
+
+nasgateway:
+  enabled: false
+  # Number of parallel instances
+  replicas: 4
+  # Generally for NAS Gateway, you'd like to bind the PVC to a specific PV. To ensure that happens, PV to bind to should have
+  # a label like "pv: <value>", use value here.
+  pv: ""
+  # Storage class of PV to bind. Be default it looks for standard storage class. If the PV uses a different storage class, specify
+  # that here.
+  storageClass: standard
```

Comment:
Why don't you simply use `persistence.storageClass`? You could avoid the conditional logic.